### PR TITLE
feat(icons): add new icons

### PR DESCRIPTION
**Pull request by matoous**
## Purpose

Add new icons.

## Approach and changes

Adds new icons and sorts the icon manifest file alphabetically.

- **feat(icons): add new icons**
- **sort manifest**

## Definition of done

* [ ] Development completed
* [ ] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

File: packages/icons/web/v2/print_failed_16.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_18374_738)">
+<path d="M5 0C4.44772 0 4 0.447715 4 1V2H12V1C12 0.447715 11.5523 0 11 0H5Z" fill="currentColor"/>
+<path d="M2 4C0.895431 4 0 4.89543 0 6V11C0 11.5523 0.447715 12 1 12H2.67184L0.400076 14.2718C0.00956237 14.6623 0.00956371 15.2955 0.400079 15.686C0.785751 16.0716 1.40808 16.0764 1.79962 15.7003L15.3492 2.1507C15.3532 2.14679 15.3571 2.14291 15.361 2.13908L15.6953 1.80487C16.0858 1.41435 16.0858 0.781204 15.6953 0.390689C15.3047 0.000171363 14.6716 0.000171691 14.2811 0.39069L10.6718 4H2Z" fill="currentColor"/>
+<path d="M15.5689 4.75947L11 9.32837V13C11 13.5523 10.5523 14 10 14H6.32837L4.41521 15.9132C4.60019 15.9696 4.79655 16 5 16H10C11.6569 16 13 14.6569 13 13V12H15C15.5523 12 16 11.5523 16 11V6C16 5.53138 15.8388 5.1004 15.5689 4.75947Z" fill="currentColor"/>
+</g>
+<defs>
+<clipPath id="clip0_18374_738">
+<rect width="16" height="16" fill="white"/>
```
Comment (matoous) [inline packages/icons/web/v2/print_failed_16.svg:9]: @connor-baer the `white` here seems wrong, what's the correct approach please?

File: packages/icons/manifest.json
Comment (connor-baer) [inline packages/icons/manifest.json:1]: This file was already grouped and sorted by category first, and sorted by name second.

File: packages/icons/web/v2/print_failed_16.svg (hunk 1 above)
Comment (connor-baer) [reply packages/icons/web/v2/print_failed_16.svg]: The `clipPath` is superfluous; this is described in more detail in the [docs](https://circuit.sumup.com/?path=/docs/contributing-contributing-icons--docs#beware-clip-path). 

I've pushed a commit to remove it.

File: packages/icons/manifest.json
Comment (matoous) [reply packages/icons/manifest.json:1]: Ah damn, will take a look if I can replicate that without doing that manually. Do you happen to know how was it done?

Comment (connor-baer) [reply packages/icons/manifest.json:1]: That's very neat, thank you!

**Review by connor-baer — APPROVED** [review]
Going above and beyond, as always 🙌

**Pull request merged**